magento/adobe-stock-integration
# Show error when user is not able to login && Show image file name validation message in the confirmation popup

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#734: I Show error when user is not able to login
2. magento/adobe-stock-integration#766:Show image file name validation message in the confirmation popup

### Manual testing scenarios (*)

1.     Go to Stores -> Configuration -> System -> Adobe Stock Integration fieldset
2.     Enter valid API Key (Client ID)
3.     Enter incorrect Client Secret
4.     Save Configuration
5.     Open Adobe Stock Integration Grid

    6a. Login to Adobe Stock using "Sign in" button in top right corner
    6b. Initiate login by trying to license any image

6a: User is not able to log in. A corresponding error is shown to the user in a popup
6b: User is not able to log in. A corresponding error is shown to the user above the action buttons in the image preview
Error message: “Login failed. Check if the Secret Key</link to the configuratioon> entered correctly and try again.”
